diesel/src/query_builder/functions.rs

```diff
@@ -146,9 +161,14 @@ pub fn delete<T: IntoUpdateTarget>(source: T) -> DeleteStatement<T::Table, T::Wh
     DeleteStatement::new(target.table, target.where_clause)
 }
 
-/// Creates an insert statement. Will add the given data to a table. This
-/// function is not exported by default. As with other commands, the resulting
-/// query can return the inserted rows if you choose.
+/// Creates an `INSERT`  statement.
```

Comment:
Empty doc comment line below this on. Also, one space should be enough after "`INSERT`" :)